teradatalabs/mapr52-hive/files/wardenTracker.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
I don't know enough bash to say if this is idiomatic so I'll leave the review here to @ArturGajowy.